jsonrpc/eth_endpoint.go

```diff
@@ -717,7 +717,7 @@ func (e *Eth) GetTransactionCount(address types.Address, filter BlockNumberOrHas
 
 	// The filter is empty, use the latest block by default
 	if filter.BlockNumber == nil && filter.BlockHash == nil {
-		filter.BlockNumber, _ = createBlockNumberPointer("latest")
+		filter.BlockNumber, _ = createBlockNumberPointer(latest)
```

Comment:
Not sure why error is discarded (also in the `helper.go`: `GetHeaderFromBlockNumberOrHash`).

Reply:
The error can not be raised if you call this function with latest.